com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -215,6 +215,7 @@ static void OnBakeStarted()
         static void OnAdditionalProbesBakeCompleted()
         {
             UnityEditor.Experimental.Lightmapping.additionalBakedProbesCompleted -= OnAdditionalProbesBakeCompleted;
+            UnityEngine.Profiling.Profiler.BeginSample("OnAdditionalProbesBakeCompleted");
```

Comment:
Do we want to keep those or was it just for testing purpose during developpement?

Reply:
I think it wouldn't hurt keeping this here.

Reply:
I think it got removed at some point and a merge conflict included it back, I can keep it :)